packages/infrastructure/kube_ingress/vars.tf

```diff
@@ -127,6 +127,18 @@ variable "rate_limiting_enabled" {
   default     = true
 }
 
+variable "connection_timeout_seconds" {
```

Comment:
This should not be named `connection_timeout_seconds`. It should be `idle_timeout_seconds` because what it is doing is setting the timeout that NGINX will wait between receiving and writing packets of data before determining that the connection is dead.

`connection_timeout_seconds` sort of implies `connect_timeout_seconds` which is a completely different concept.

See more [here](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_send_timeout)